R/roxygen2.R

```diff
@@ -17,6 +17,39 @@ roxygen_param_dataset <- function(expected_vars = NULL) {
   return(dataset_text)
 }
 
+
+# function to properly document the by_vars argument including specifying the use of exprs():
+
+roxygen_param_by_vars <- function(additional_dataset = NULL, unique = FALSE, rename = FALSE) {
+  by_vars_text <- ""
+
+  if (!is.null(additional_dataset)) {
+    if (unique) {
+      by_vars_text <- paste0(
+        by_vars_text,
+        "Variables must be a unique key of the selected observations in ",
+        enumerate(additional_dataset), ". \n \n"
+      )
+    }
+    if (rename) {
+      by_vars_text <- paste0(
+        by_vars_text,
+        "Variables from ", enumerate(additional_dataset),
+        " can be renamed by naming the element, I.e. \n",
+        "by_vars = exprs(<name in input dataset> = <name in additional dataset>),",
```

Comment:
Backticks should be used. Otherwise, it looks quite strange:
![image](https://github.com/pharmaverse/admiral/assets/80953585/461d291a-20c6-46a0-8153-a320979e1b65)